README.md

```diff
@@ -1 +1,59 @@
-# Salama-Conference-Capstone-Project
+
+
+# Portfolio-Site
+-This is my capstone project at Microverse as a beginner software developer. It's a conference page to speak about Gender-Based-Violence and Sexual Abuse for better treatment of victims and reducing abuse cases. The project utilized a design by Cindy Shin.
+
+## Built With
+
+- Major languages 
+  HTML | CSS | Javascript
+- Frameworks
+  None
+- Technologies used
+  Nodejs, jQuery
+  
+ ### Prerequisites
+HTML, CSS
+
+### Getting Started
+To get a local copy for practice or as a template, follow these simple steps.
+
+Access the live source on - https://github.com/sharon-odhiambo/Salama-Conference-Capstone-Project
+Copy the **https** or **SSH** address on the code
+Clone the repository using your editor    **or**
+
+### Clone this repository
+$ git@github.com:sharon-odhiambo/Salama-Conference-Capstone-Project.git
+$ cd My-Portfolio-Site
+
+
+### Live Demo
+
+Live Version - https://sharon-odhiambo.github.io/
+
+## Author
+
+👤 **Sharon Odhiambo**
+
+- GitHub: [@sharon-odhiambo](https://github.com/sharon-odhiambo)
+- Twitter: [@SharonVictor16](https://twitter.com/SharonVictor16)
+- LinkedIn: [Sharon Odhiambo](https://www.linkedin.com/in/sharon-odhiambo-4333a0163/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
```

Comment:
Please give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). 😃